File: 仿QQ侧滑视图/仿QQ侧滑视图/ViewController.m
//
//  ViewController.m
//  仿QQ侧滑视图
//
//  Created by BoBo on 17/2/7.
//  Copyright © 2017年 Li Sen. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (nonatomic, strong) UIPanGestureRecognizer *rightPanGes;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.view.backgroundColor = [UIColor yellowColor];
    self.rightPanGes = [[UIPanGestureRecognizer alloc] initWithTarget:self action:@selector(UIPanGestureDidMoved)];
    [self.view addGestureRecognizer:self.rightPanGes];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

#pragma event response
- (void)UIPanGestureDidMoved {
    NSLog(@"........");
}


@end

File: App之间的跳转/App之间的跳转/ViewController.swift
//
//  ViewController.swift
//  App之间的跳转
//
//  Created by BoBo on 2017/4/11.
//  Copyright © 2017年 Li Sen. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        view.backgroundColor = .red
        //在info.plist中添加 URL Types,添加URL Scheme 和 URL Identifier
        //在Safari中输入URL Scheme://即可完成App的跳转
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

